instance/tests/integration/integration_instance.py

```diff
@@ -106,6 +143,8 @@ def test_external_databases(self):
         self.assert_instance_up(instance)
         self.assertTrue(instance.successfully_provisioned)
         self.assertFalse(instance.require_user_creation_success())
+        self.assert_mysql_db_provisioned(instance)
+        self.assert_mongo_db_provisioned(instance)
```

Comment:
That is correct. We're making the connections from the integration-spawned OCIM, so we shouldn't be able to make connections to appserver-internal databases.